paritytech/parity-signer
# feat: Add westend network support

closes #540 closes #466

This PR has essential update for the release of Polkadot Network.

### Add Westend Support

Signing extrinsics have been test on iOS with Polkadot.js.

### Upgrade @polkadot.js related dependencies and React Native

`__dirname` is used across @polkadot packages, to prevent the version conflict of client side app and browser extension, here I have a custom babel plugin to rewrite all the `__dirname` into static path string value as a walkaround.

### How to test:

1. Create new network with `Westend` network, and create any account under it.
2. Signing extrinsics works for any account under `Westend` network.
3. At the same time, signing for kusama account should not be broken.
